### PR TITLE
[WIFI-2920] Add: helm chart for deployment

### PR DESCRIPTION
**Pull request by dunaev-opsfleet**

### CODE REVIEW

File: helm/values.yaml [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+  enabled: true
+  # storageClassName: "-"
+  accessModes:
+    - ReadWriteOnce
+  size: 10Gi
+  annotations: {}
+
+# Application
+public_env_variables:
+  UCENTRALSEC_ROOT: /ucentral-data
```
Comment (oblom0v) [inline helm/values.yaml:104]: Should we change this to `/ucentralsec-data`? At least this directory is created in the Dockerfile.

Comment (dunaev-opsfleet) [reply helm/values.yaml]: Done

File: helm/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+      containers:
+
+        - name: ucentralsec
+          image: "{{ .Values.images.ucentralsec.repository }}:{{ .Values.images.ucentralsec.tag }}"
+          imagePullPolicy: {{ .Values.images.ucentralsec.pullPolicy }}
+
+          command:
+            - /ucentral/ucentralsec
+            - --file=/ucentral-data/ucentralsec.properties
```
Comment (oblom0v) [inline helm/templates/deployment.yaml:43]: Has to be changed to `/ucentralsec-data/ucentralsec.properties` at this point. But do we need to append this option? Should be the default anyway as far as I know.

Comment (dunaev-opsfleet) [reply helm/templates/deployment.yaml:43]: Yes, good notice

Comment (dunaev-opsfleet) [reply helm/templates/deployment.yaml]: Fixed, also fixed in ucentralsec

**Review by oblom0v — APPROVED** [review]

**Pull request merged**